facebook/app/controllers/home_controller.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+class HomeController < ApplicationController
+  before_action :set_post, only: %i[show edit update destroy]
+  # GET /posts
+  # GET /posts.json
+  def index
+    @posts = Post.all
```

Comment:
Here, things should change. You must no longer take all the posts but only created by the current user or by his friends